schedule/yam/agama_auto_add_user.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
we can reuse the yamls created for agama_auto from previous task, adding user is a default.